packages/api/config/index.ts

```diff
@@ -18,7 +18,7 @@ export const getConfig: () => Promise<AppConfig> = async () => {
   // Load any ENV vars from local .env.local file
   if (process.env.NODE_ENV !== "production") {
     console.warn("Loading environment variables from root '.env.local' file. THIS SHOULD NOT BE USED IN PRODUCTION!");
-    dotenv.config({ path: path.resolve(process.cwd(), "../../.env.local") });
+    dotenv.config({ path: path.resolve(process.cwd(), ".env.local") });
```

Comment:
I am afraid this change will break the API when running everything from the root of this monorepo. We need to think of a better flow to support both scenarios.

Reply:
@manekinekko the location in the api package.json for the 'env' script doesn't match the config/index.ts location. The [ed7f13b](https://github.com/Azure-Samples/contoso-real-estate/pull/344/commits/ed7f13b480a0b24a090cc8f2436521a7497dff71) commit fixes this.